src/Common/src/TypeSystem/Common/MetadataVirtualMethodAlgorithm.cs

```diff
@@ -503,8 +503,12 @@ private static MethodDesc ResolveInterfaceMethodToVirtualMethodOnType(MethodDesc
                 if (baseType == null)
                     return foundOnCurrentType;
 
-                if (foundOnCurrentType == null && (ResolveInterfaceMethodToVirtualMethodOnType(interfaceMethod, baseType) == null))
```

Comment:
This change is incorrect. See the comment above the method about how order of discovery is supposed to affect variant interfaces. As this change didn't cause a unit test failure, we really should add such a unit test so that we don't incorrectly make this change again.